src/worker/storage.ts

```diff
@@ -2,7 +2,7 @@ import Dexie, { DexieOptions } from "dexie";
 import { WordList } from "@common/types";
 
 const DB_NAME = "dictionary_sources";
-const SCHEMA_VERSION = 1;
+const SCHEMA_VERSION = 2;
```

Comment:
I think we can get away with this by just deleting the Indexed DB in the browser

Reply:
you mean I can keep the version = 1?

Reply:
right, since there are no actual users at the moment

Reply:
But when I changed the version number to 1 and downloaded the kmp file, bcp47Tag didn't update, it still used the initial value "".

Reply:
You need to delete the old indexed DB, I can get the new tag after I have done that
```
[{"name":"English dictionary","id":"unknownauthor.aa.example","languages":[{"name":"English","id":"aa"}]}]}
```

Reply:
You're right!

Reply:
I'd like to get in the habit of incrementing the version ID every time we make a change to the schema, because "just delete your indexedDB" is not always going to be an option!